packages/calcite-components/src/components/list/list.e2e.ts

```diff
@@ -530,6 +543,35 @@ describe("calcite-list", () => {
     expect(await list.getProperty("selectedItems")).toHaveLength(0);
   });
 
+  it("should show noFilterResults", async () => {
```

Comment:
Nitpick: `noFilterResults` reads like a prop and does not match the slot name. WDYT about `"should show no-results content when filter does not match"`?